foxy/distribution.yaml

```diff
@@ -2550,6 +2550,24 @@ repositories:
       url: https://github.com/ros-planning/random_numbers.git
       version: ros2
     status: maintained
+  raptor_dbw_ros2:
+    release:
+      packages:
+      - raptor_can_dbc_parser
+      - raptor_dbw_can
+      - raptor_dbw_joystick
+      - raptor_dbw_msgs
+      - raptor_pdu
+      - raptor_pdu_msgs
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/NewEagleRaptor/raptor-dbw-ros2-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/NewEagleRaptor/raptor-dbw-ros2.git
+      version: foxy-1.0.0
```

Comment:
Tags are not valid for source (or doc) entries. Instead point to the branch associated with the released code (e.g. `foxy`).

This should fix CI.

Reply:
Fixed (I'm helping @NewEagleRaptor).

Reply:
Yep, we appreciate it!